test/core/utils/get-selector.js

```diff
@@ -551,6 +551,16 @@ describe('axe.utils.getSelector', function () {
 		);
 	});
 
+	it('should work correctly when a URL attribute cannot be shortened', function () {
+		var node = document.createElement('a');
+		node.setAttribute('href', 'mars2.html?a=be_bold');
```

Comment:
It'd be nice to cache "mars2.html?a=be_bold" so it doesn't have to be repeated throughout the test. It would be less prone to typos.